lib/pyenv.rb

```diff
@@ -18,16 +18,18 @@ def initialize(build_wrapper)
 
     def setup!
       install!
+      update!
```

Comment:
I think moving `update!` here will make all builds to update pyenv. I guess do something like following can make pyenv's update fewer as possible. How do you think about this?

```rb
available_versions = capture(pyenv("versions", "--bare")).strip.split
unless version.split(":").all? { |version| available_versions.include?(verion) }
  update!
  version.split(":").each do |version|
    unless versions.include?(version)
      listener << "Installing #{version}..."
      run(pyenv("install", version), {out: listener})
      listener << "Installed #{version}."
    end
  end
end
```